kernel/arch/dreamcast/hardware/pvr/pvr_send_to_ta.s

```diff
@@ -0,0 +1,43 @@
+! KallistiOS ##version##
+!
+!   arch/dreamcast/hardware/pvr/pvr_send_to_ta.s
+!   Copyright (C) 2024 Paul Cercueil
+!
+! Fast function to upload geometry to the Tile Accelerator without SQs
+!
+
+.globl _pvr_send_to_ta
+
```

Comment:
```suggestion

!
! void pvr_send_to_ta(void *data);
!
! r4: data (should be 8-byte aligned address)
!
```

Reply:
Not sure if this adds anything or not but I think it could be helpful to some who are just starting out looking at SH4 assembly. Feel free to disregard.